dagmc_stats.py

```diff
@@ -3,23 +3,21 @@
 sys.path.append('/opt/tljh/user/lib/moab/lib/python3.6/site-packages/pymoab-5.1.0-py3.6-linux-x86_64.egg')
 from pymoab import core, types
 from pymoab.rng import Range
-
 def get_dagmc_tags(my_core):
     """
     Get a dictionary with the important tags for DAGMC geometries
-
+    
     inputs
     ------
     my_core : a MOAB Core instance
-
     outputs
     -------
     dagmc_tags : a dictionary of relevant tags
     """
 
     dagmc_tags = {}
     dagmc_tags['geom_dim'] = my_core.tag_get_handle('GEOM_DIMENSION', size = 1, tag_type = types.MB_TYPE_INTEGER, #creates tag for the geo-
-                                                 storage_type = types.MB_TAG_SPARSE, create_if_missing = True)# metric dimension
+                                                 storage_type = types.MB_TAG_SPARSE, create_if_missing = True)# geometric dimension
```

Comment:
Those line look very long. 
PEP8 (our reference style guide for python development) suggest 80 char per line. (there is tools allowing to auto format code to pep8 requirement (python package called `autopep8`). it is an easy way to clean the style of your the code

Reply:
you can find docs on pep8 [[here](https://www.python.org/dev/peps/pep-0008/)]